routes/warehouse.js

```diff
@@ -23,5 +23,7 @@ router.use('/:id/inventories', async (req, res)=>{
     }
 });
 
+router
```

Comment:
Not sure if i'm seeing this right but here should start with: router.route(path).delete(etc)